pkg/controller/certificates/signer/signer.go

```diff
@@ -305,3 +315,152 @@ func usagesToSet(usages []capi.KeyUsage) sets.String {
 	}
 	return result
 }
+
+type PCRSigningController struct {
+	pcrController       *certificates.PodCertificateRequestController
+	dynamicCertReloader dynamiccertificates.ControllerRunner
```

Comment:
Using `DynamicFileCAContent` instead would allow you to hook CA-changed events into the `pcrController` and promptly react to signer rotations.

Reply:
Ditto, this was dropped from this PR.